docs/source/user_guide/pipeline-components.md

```diff
@@ -166,6 +166,46 @@ $ elyra-metadata remove component-registries \
 
 Refer to section [Configuration properties](#configuration-properties) for parameter descriptions.
 
+#### Migrating user-defined component registries to 3.3
```

Comment:
Should we promote this section to H3 from H4 to make the migration topic appear when the user expands the `pipeline components` topic? As is, the migration content is easy to miss unless one browses the entire topic or drills down on the CLI section.

![image](https://user-images.githubusercontent.com/13068832/141874039-07ab5d03-f0ae-4f74-8f1c-bb4fa19413b7.png)